options/ansi/generic/fenv-stubs.cpp

```diff
@@ -2,54 +2,39 @@
 #include <bits/ensure.h>
 #include <fenv.h>
 
-int feclearexcept(int) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
-}
-
-int fegetenv(fenv_t *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
-}
+// The functions that are not in this file but are defined in the header
+// are implemented like musl does in assembly.
+extern "C" __attribute__((__visibility__("hidden"))) int __fesetround(int);
```

Comment:
Isn't making the assembly hidden sufficient? Also, you could use C++-style attributes

Reply:
Nvm, disregard. But C++ attribute would be nice

Reply:
Couldn't get it to work, so we can fix that later if need be.